pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/QueryRunner.java

```diff
@@ -175,14 +175,15 @@ public void init(PinotConfiguration config, InstanceDataManager instanceDataMana
     String joinOverflowModeStr = config.getProperty(CommonConstants.MultiStageQueryRunner.KEY_OF_JOIN_OVERFLOW_MODE);
     _joinOverflowMode = joinOverflowModeStr != null ? JoinOverFlowMode.valueOf(joinOverflowModeStr) : null;
 
-    _executorService = ExecutorServiceUtils.create(config, Server.MULTISTAGE_EXECUTOR_CONFIG_PREFIX,
-        "query-runner-on-" + port, Server.DEFAULT_MULTISTAGE_EXECUTOR_TYPE);
-
-    int hardLimit = HardLimitExecutor.getMultiStageExecutorHardLimit(config);
```

Comment:
Where was this block of code moved to ?

Reply:
It was deleted by mistake while resolving a conflict during a merge. It is fixed in the last commit